src/components/KeyStats.js

```diff
@@ -1,10 +1,11 @@
 import React from "react";
 import { useSelector } from "react-redux";
+import { Loading } from "./Loading";
 
 const NUMBER_FORMATTER = new Intl.NumberFormat();
 
 const DEFAULT_FORMATTER = (data, key) =>
-  data[key] != undefined ? data[key] : "N/A";
+  data[key] !== undefined && data[key] !== null ? data[key] : "N/A";
```

Comment:
How come you expanded this out, rather than sticking with `!=`?

Reply:
because it always shows the error when running it

Reply:
In JS, the previous code and the current code are functionally identical.
![image](https://user-images.githubusercontent.com/38944003/66758931-2b926e80-ee97-11e9-8ec5-d3ce84b2b9f2.png)